tests/test_api_auth.py

```diff
@@ -348,10 +331,6 @@ def test_02_auth_without_split(self):
                 result = res.json.get("result")
                 self.assertFalse(result.get("status"), result)
                 self.assertEqual(4031, result['error']['code'], result)
-            if six.PY2:
-                expected = "The user User(login=u'selfservice@realm3', " \
-                           "realm=u'realm1', resolver='') exists in NO resolver."
-            else:
                 expected = "The user User(login='selfservice@realm3', " \
```

Comment:
here also

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.